.dev/README.md

```diff
@@ -1,5 +1,14 @@
 # data.table developer
 
+## Setup
+
+One needs to set up the project path and source `.dev/cc.R` to use `cc()` conveniently. This works through creating an additional `.Rprofile` in the `data.table` directory.
```

Comment:
It feels like this is required for developing DT. It is a optional helper function so should not sound like a requirement for setting up Dev environment.

Reply:
I agree. A quick change to: "An optional helper function can be accessed by setting up the project path and source ..."